### DISCUSSION TITLE
Update with new OGN version during operation

### DISCUSSION THREAD

**Discussion by LogX788** [VirusPilot/ogn-pi34#9] [General]
Firstly, I would like to thank you for your great work. I am currently running your version with OGN 3.1. How can I update to 3.2. without having to set up the whole RPi again. 
Best regards

**Comment by VirusPilot** [comment]
Sure, happy that it works. Here are the instructions: https://github.com/VirusPilot/ogn-pi34?tab=readme-ov-file#steps-to-manually-upgrade-legacy-platforms. Please carefully select the appropriate version for your platform and the location of your current version.
